### PR TITLE
WIP adds a new layer of security to bans.

### PR DESCRIPTION
**Pull request by corp-0**
# Pull Request Template

### Purpose
This PR is an attempt to add a new layer of security to bans, using the machine unique ID and adding them to ban data.

### Notes:
Currently WIP as we need to test different machines and OS's

### Please make sure you have followed the self checks below before submitting a PR:

- [ ] Code is sufficiently commented
- [ ] Code is indented with tabs and not spaces
- [ ] The PR does not include any unnecessary .meta, .prefab or **.unity (scene) changes**
- [ ] The PR does not bring up any new compile errors
- [ ] The PR has been tested in editor
- [ ] Any new files are named using PascalCase (to avoid issues on case sensitive file systems)
- [ ] Any new / changed components follow the [Component Development Checklist](https://github.com/unitystation/unitystation/wiki/Component-Development-Checklist)
- [ ] Any new objects / items follow the [Creating Items and Objects Guide](https://github.com/unitystation/unitystation/wiki/Creating-Items-and-Objects%3A-Now-With-Prefab-Variants) (especially concerning the use of prefab variants)
- [ ] The PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)
- [ ] The PR has been tested with round restarts.
- [ ] The PR has been tested on moving / rotating / rotated-before-joining matrices (if applicable)

### CODE REVIEW

**Comment by RedlineTriad** [comment]
Will test on staging.

**Comment by daalta** [comment]
Sounds like a good idea to keep track of the device id!

**Comment by daalta** [comment]
Um, isn't this still WIP @RedlineTriad ? Hope @corp-0 is OK with the merge 😅

**Comment by corp-0** [comment]
we merging to test on staging

**Comment by daalta** [comment]
Ah, you got it all planned out. Got it :)

File: UnityProject/Assets/Scripts/Managers/ConnectedPlayer.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class ConnectedPlayer
 	private NetworkConnection connection;
 	private string clientID;
 	private string userID;
+	private string deviceID = SystemInfo.deviceUniqueIdentifier;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/ConnectedPlayer.cs:16]: Problem is probably here. It works in editor but not in our server.

**Pull request merged**